app/src/main/res/values/strings.xml

```diff
@@ -335,6 +335,7 @@
     <plurals name="label_system_message_x_more">
         <item quantity="other">%1$d more</item>
     </plurals>
+    <string name="default_deleted_username">Default</string>
```

Comment:
I think it should  be "Deleted User"

Reply:
Let me check with Wolfgang, since in the old app says "default" I just grabbed from there 😅  thanks for pointing this out :D

Reply:
I just remember that we already have one that's used when the user is deleted:
```<string name="member_name_deleted_label">Deleted User</string>```
😉